indigo/bulk_creator.py

```diff
@@ -93,10 +96,63 @@ def get_doctypes_for_country(self, country_code):
                 settings.INDIGO['DOCTYPES'] +
                 settings.INDIGO['EXTRA_DOCTYPES'].get(country_code, [])]
 
+    def sanitize_incoming(self, data):
```

Comment:
A more Django-esque approach to this would be to do this in `clean_<field>`, or tweak the form widget: https://stackoverflow.com/questions/24313201/convert-all-charfield-form-field-inputs-to-lowercase-in-django-forms

But it doesn't matter that much.

Reply:
`clean_x` only gets called it if the form is valid?, and because the choices are case-sensitive it gets rejected before we get here.